source/help/getting-started/messaging-basics.rst

```diff
@@ -3,9 +3,15 @@ Messaging Basics
 
 --------------
 
+.. |Main menu icon| image:: ../../images/sidebar_main_menu_icon.png
```

Comment:
Thanks @amaddio, propose we remove the icon image given it's inconsistent with the rest of our documentation, if you're okay with that?

Reply:
You are right. It is called `Main Menu` on other pages and also defined e.g. [account settings page](https://docs.mattermost.com/help/settings/account-settings.html?highlight=main%20menu#send-messages-on-ctrl-enter). So let's get rid of the icon.